# 적용 순서: Roll(X) -> Pitch(Y) -> Yaw(Z)

transforms:
  # base_link -> bev
  - parent: base_link
    child: bev
    translation: [0.38, 0.0, 0.30]   # [m]
    rotation_rpy_deg: [180.0, 0.0, -90.0]
  # base_link -> camera_optical_frame
  - parent: base_link
    child: camera_optical_frame
    translation: [0.28, 0.00, 0.15]
    rotation_rpy_deg: [-90.0, 0.0, -90.0]
  # # base_link -> 2d lidar
  # - parent: base_link
  #   child: laser
  #   translation: [0.20, 0.00, 0.15]
  #   rotation_rpy_deg: [0.0, -10.0, 0.0]
  # # base_link -> imu
  # - parent: base_link
  #   child: imu
  #   translation: [0.20, 0.00, 0.15]
  #   rotation_rpy_deg: [0.0, -10.0, 0.0]

